Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridCellsPresenter.cs

```diff
@@ -274,9 +274,6 @@ protected override Size MeasureOverride(Size availableSize)
             // then we will resize all the columns to fit the available space.
             if (this.OwningGrid.UsesStarSizing && !this.OwningGrid.AutoSizingColumns)
             {
-                double adjustment = this.OwningGrid.CellsWidth - totalDisplayWidth;
```

Comment:
Why was this removed? It's part of the loop, so this calculation should matter?

Reply:
We are no longer in the loop but my nitial understanding of `AdjustColumnWidths()` was incorrect. We still need to call it since it mutate the state of the data grid. I will be less aggressive and remove only the update of `totalDisplayWidth` which is no needed.

---
In reply to: [390620187](https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3155#discussion_r390620187) [](ancestors = 390620187)